.eslintrc.yaml

```diff
@@ -135,16 +135,17 @@ rules:
   }]
   no-tabs: error
   no-trailing-spaces: error
+  object-curly-spacing: [error, always]
```

Comment:
In general it's useful to do the automatic ones as one commit, and then the manual ones as a separate one. They'll get squashed on landing, but it means people can review the ones that weren't just an `eslint --fix` (which are probably the more interesting ones).